src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -13,7 +14,6 @@ interface TransactionHistoryCardComponent {
   }[];
   loading: boolean;
 }
-
```

Comment:
Nit, please revert this. So only changes related to this feature is reflected in the version history.

Reply:
done